src/js/components/HealthBar.js

```diff
@@ -0,0 +1,78 @@
+import classNames from 'classnames';
+import React from 'react';
+import {Tooltip} from 'reactjs-components';
+
+import HealthBarMap from '../constants/HealthBar';
+import StatusBar from './StatusBar';
+
+class HealthBar extends React.Component {
+  static get defaultProps() {
+    return {
+      instancesCount: null
+    };
+  }
+
+  renderToolTip() {
+    let {tasks, instancesCount} = this.props;
+
+    tasks = Object.keys(tasks).filter(function (task) {
+      return tasks[task] !== 0 && task !== 'tasksRunning';
+    }).map(function (task, index) {
+      let percentage = parseInt(tasks[task] / instancesCount * 100, 10);
+
+      let classSet = classNames(HealthBarMap[task].cssName, 'dot icon');
+
+      return (
+        <div key={index}>
+          <span className={classSet}></span>
+          {` ${tasks[task]} ${HealthBarMap[task].label} (${percentage} %)`}
+        </div>
+      );
+    }).filter(function (item) {
+      return item != null;
+    });
+
+    if (tasks.length === 0) {
+      return (
+        <div>
+          No Running Tasks
+        </div>
```

Comment:
Worked I added that to the other PR where this feature is introduced. I will rebase this branch to `feature/unified-ui` so that we do not get mixed up in the commits.